streams/hash/others/hash_functions/hash_functions.h

```diff
@@ -0,0 +1,5 @@
+#pragma once
```

Comment:
I don't like this file so much, I think this should be in factory, but if you like this more why not. Now I realized we have this also in sha3. I think we should stay consistent with blocks and estream and put all includes into factory file.